File: backend/apps/usuarios/routes/profesional_routes.py
from django.urls import path

from ..controllers.profesional_controller import (
    CrearProfesionalView,
    ClientesProfesionalView,
)

urlpatterns = [
    path("profesionales/", CrearProfesionalView.as_view(), name="crear-profesional"),
    path("profesional/<int:profesional_id>/clientes/", ClientesProfesionalView.as_view(), name="clientes-profesional"),
]

File: backend/src/routes/profesional.routes.ts
import { Router } from "express";
import { ProfesionalController } from "../controllers/profesional.controller";
import { authenticateToken } from "../middlewares/auth.middleware";
import { authorizeRoles } from "../middlewares/role.middleware";
import { ROLES } from "../data/roles";

const router = Router();
router.get("/profesionales/", ProfesionalController.listar);
router.post("/profesionales/", authenticateToken, authorizeRoles(ROLES.PRO, ROLES.ADMIN), ProfesionalController.crear);
router.get("/profesional/:profesionalId/clientes/", authenticateToken, authorizeRoles(ROLES.PRO, ROLES.ADMIN), ProfesionalController.clientes);
export default router;
